commands/help.js

```diff
@@ -28,5 +28,5 @@ module.exports = new Command("help [command]")
       logger.info();
     }
 
-    RSVP.resolve();
+    Promise.resolve();
```

Comment:
does this need to be `return Promise.resolve()`?

Reply:
Done.